src/python/pants/option/global_options.py

```diff
@@ -1051,6 +1061,17 @@ def register_bootstrap_options(cls, register):
             help="Number of times to retry any RPC to the remote store before giving up.",
         )
 
+        register(
+            "--remote-cache-warnings",
+            type=RemoteCacheWarningsBehavior,
+            default=DEFAULT_EXECUTION_OPTIONS.remote_cache_warnings,
+            advanced=True,
+            help=(
+                "Whether to log remote cache failures at the `warn` log level.\n\n"
+                "All errors not logged at the `warn` level will instead be logged at the "
+                "`debug` level."
```

Comment:
I debated explaining what option does, but `ignore` and `first_only` seem pretty self-explanatory. `backoff` could maybe use an explanation, although I doubt we want to mention the implementation detail of using the exponent of 2.